editor/vscode-client/src/extension.ts

```diff
@@ -0,0 +1,65 @@
+import * as path from 'path';
+import { workspace, ExtensionContext } from 'vscode';
+
+import {
+	Executable,
+	LanguageClient,
+	LanguageClientOptions,
+	ServerOptions,
+} from 'vscode-languageclient/node';
+
+let client: LanguageClient;
+
+export function activate(context: ExtensionContext) {
+	const binPath: string = workspace.getConfiguration('jsonnet').get('languageServer.pathToBinary');
+
+	if (binPath === null) {
+		// TODO: Auto-install latest released version when pathToBinary is not set

```

Comment:
Nice! I'm playing around with Snitch for managing todos and GitHub Issues in this project. Would you mind rephrasing this so it has enough context to stand as an Issue.

```suggestion
		// TODO: vscode-client should auto-install the latest released version when pathToBinary is not set
```
or something similar